lib/actions-oclif/key/rm.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import { stripIndent } from 'common-tags';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk } from '../../utils/lazy';
+import { parseAsInteger } from '../../utils/oclif-validation';
+
+type IArg<T> = import('@oclif/parser').args.IArg<T>;
+
+interface FlagsDef {
+	yes: boolean;
+	help: void;
+}
+
+interface ArgsDef {
+	id: number;
+}
+
+export default class KeyRmCmd extends Command {
+	public static description = stripIndent`
+		Remove an SSH key.
+
+		Remove a single SSH key from balena.
+
+		Notice this command asks for confirmation interactively.
+		You can avoid this by passing the \`--yes\` boolean option.
```

Comment:
'Note' sounds better than 'Notice'? Also personally I prefer to avoid the pronoun "You". Suggestion: instead of the two lines above (from 'Notice' to 'option'):
```
The --yes option may be used to avoid interactive confirmation.
```